src/app/public/modules/colorpicker/colorpicker.component.html

```diff
@@ -1,186 +1,236 @@
 <div
   class="sky-input-group"
-  [ngClass]="{ 'sky-colorpicker-hidden': !isVisible }">
-  <ng-content>
-  </ng-content>
-  <sky-dropdown
-    [label]="'skyux_colorpicker_dropdown_button' | skyLibResources"
-    [messageStream]="dropdownController"
-    [dismissOnBlur]="false">
-    <sky-dropdown-menu>
-      <div class="sky-colorpicker-container">
+>
+  <div
+    [hidden]="true"
+  >
+    <ng-content></ng-content>
+  </div>
+
+  <button
+    aria-haspopup="true"
+    class="sky-colorpicker-button"
+    type="button"
+    [attr.aria-controls]="colorpickerId"
+    [attr.aria-expanded]="isOpen"
+    [attr.aria-label]="'skyux_colorpicker_dropdown_button' | skyLibResources"
+    [attr.id]="triggerButtonId"
+    [attr.title]="'skyux_colorpicker_dropdown_button' | skyLibResources"
+    [style.background-color]="backgroundColorForDisplay"
+    (click)="onButtonClick()"
```

Comment:
Since you're calling this a `triggerButton` elsewhere...should this be `onTriggerButtonClick()`?